Mathlib/AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Basic.lean

```diff
@@ -272,20 +282,30 @@ lemma ΨSq_three : W.ΨSq 3 = W.Ψ₃ ^ 2 := by
 lemma ΨSq_four : W.ΨSq 4 = W.preΨ₄ ^ 2 * W.Ψ₂Sq := by
   erw [ΨSq_ofNat, preΨ'_four, if_pos <| by decide]
 
-lemma ΨSq_odd (m : ℕ) : W.ΨSq (2 * (m + 2) + 1) =
-    (W.preΨ' (m + 4) * W.preΨ' (m + 2) ^ 3 * (if Even m then W.Ψ₂Sq ^ 2 else 1) -
-      W.preΨ' (m + 1) * W.preΨ' (m + 3) ^ 3 * (if Even m then 1 else W.Ψ₂Sq ^ 2)) ^ 2 := by
-  erw [ΨSq_ofNat, preΨ'_odd, if_neg (m + 2).not_even_two_mul_add_one, mul_one]
-
-lemma ΨSq_even (m : ℕ) : W.ΨSq (2 * (m + 3)) =
+lemma ΨSq_even' (m : ℕ) : W.ΨSq (2 * (m + 3)) =
```

Comment:
Can you add some doc expaining why this theorem has a primed name?

Reply:
I will append `_ofNat` to these.

Reply:
Yes, overall having a descriptive naming (like `_ofNat`) seems more appropriate than primes.